data/ecosystems/i/iotex.toml

```diff
@@ -201,18 +168,6 @@ url = "https://github.com/iotexproject/iips"
 [[repo]]
 url = "https://github.com/iotexproject/ins-contracts"
 
-[[repo]]
-url = "https://github.com/iotexproject/iopay-desktop"
-missing = true
-
-[[repo]]
-url = "https://github.com/iotexproject/iopay-extension"
-missing = true
-
-[[repo]]
-url = "https://github.com/iotexproject/iopay-extension-wallet"
-missing = true
-
```

Comment:
Please keep the missing attributes since we use these for historical reasons.

Reply:
@jubos Just to make sure I understand: I removed these since they were not relevant. So do you want me to add them back with the "missing" attributes?

Reply:
Yes, because they existed at one time in case.

Reply:
@jubos thank you! All set on my end!